crates/turborepo-globwatch/src/lib.rs

```diff
@@ -214,7 +214,7 @@ impl GlobWatcher {
                                 // requester. flushes should not be considered as events.
                                 for flush_id in e
                                     .paths
-                                    .extract_if(|p| p.starts_with(flush_dir.as_path()))
+                                    .extract_if(.., |p| p.starts_with(flush_dir.as_path()))
```

Comment:
I'm not sure why, but I'm pretty positive this change is resulting in [this failure](https://github.com/vercel/turborepo/actions/runs/13516449852/job/37766021434?pr=10024#step:6:2200) as this change is in that codepath and the failure seems to be happening consistently in this PR. I have not seen this test be flakey in the past.